app/people.js

```diff
@@ -23,6 +72,20 @@ class PeopleService extends Service {
     const url = this.client.url("/people/@self")
     return await this.client.request("GET", url)
   }
+
+  /**
+   * ```http
+   * GET /people
+   * ```
+   * @returns {Promise<User[]>}
+   */
+  userList() {
```

Comment:
Please be consistent. Rename this method to the `listUsers`.

Reply:
Method renamed.

Reply:
> Please be consistent. Rename this method to the `listUsers`.



Method renamed.